src/mono/System.Private.CoreLib/src/System/Threading/ThreadPool.Browser.Mono.cs

```diff
@@ -150,5 +150,8 @@ public static bool BindHandle(SafeHandle osHandle)
         {
             throw new PlatformNotSupportedException(SR.Arg_PlatformNotSupported); // Replaced by ThreadPoolBoundHandle.BindHandle
         }
+
+        [Conditional("unnecessary")]
+        internal static void ReportThreadStatus(bool isWorking) => ReportThreadStatusCore(isWorking);
```

Comment:
I'm not sure why this would be necessary now and not before. It shouldn't be necessary since worker tracking is disabled above.